data/org.freedesktop.impl.portal.InputCapture.xml

```diff
@@ -72,6 +76,86 @@
       <annotation name="org.qtproject.QtDBus.QtTypeName.Out1" value="QVariantMap"/>
       <arg type="a{sv}" name="results" direction="out"/>
     </method>
+
+    <!--
+        CreateSession2:
+        @session_handle: Object path for the :ref:`org.freedesktop.impl.portal.Session` object representing the session being created
+        @app_id: App id of the application
+        @options: Vardict with optional further information
+
+        Create a capture input session. In contrast with CreateSession, the
+        created session is not in the started state.
+
+        There are currently no supported keys in the @options vardict.
+
+        This method was added in version 2 of this interface.
+    -->
+    <method name="CreateSession2">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="s" name="app_id" direction="in"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In2" value="QVariantMap"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+    </method>
+
+    <!--
+        Start:
+        @handle: Object path for the :ref:`org.freedesktop.impl.portal.Request` object representing this call
+        @session_handle: Object path for the :ref:`org.freedesktop.portal.Session` object
+        @app_id: App id of the application
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @options: Vardict with optional further information
+        @response: Numeric response
+        @results: Vardict with the results of the call
+
+        Start the remote desktop session. This will typically result in the portal
+        presenting a dialog letting the user decide whether they want to allow the
+        input of the session to be or not, and what capabilities to support.
```

Comment:
needs rewording, looks like it was copied from remote desktop.

Reply:
I maybe did copy it , i was told when i started the (in the pr this is based on) that all that had to happen was copy the parts clipboard parts from remote desktop to input capture . When i did the discussion about new names etc started. sorry if this was incorrect .